tests/components/matter/test_light.py

```diff
@@ -48,6 +49,18 @@ async def test_on_off_light(
     assert state is not None
     assert state.state == "off"
 
+    # check the supported_color_modes
+    # especially important is the onoff light device type that does have
+    # a levelcontrol cluster present which we should ignore
+    if fixture in ("onoff-light", "onoff-light-with-levelcontrol-present"):
```

Comment:
Add a `supported_color_modes` test parameter instead of using a condition here.

Reply:
done